index.html

```diff
@@ -89,7 +91,9 @@ <h3 class="speaker-main">Featured Speakers</h3>
     <div class="speaker-list show-more">
         
     </div>
-    <button type="submit" class="more">More &nbsp; <i class="fa-light fa-angle-down"  ></i></button>
+    <div class="more">
+        More <i class="fa fa-angle-down"></i>
+    </div>
 </footer>
```

Comment:
- [ ] As requested by the previous reviewer, please make sure your footer is more centered to match the design guidelines.
 ### Design guideline
![190260226-229fe413-9393-42ff-9462-c73601eae679](https://user-images.githubusercontent.com/94551484/190514759-bc8bbd64-6740-4057-8f40-370c8467e971.png)

### Your design
![Screenshot from 2022-09-15 22-35-30](https://user-images.githubusercontent.com/94551484/190514779-62e70b3c-e939-492b-a7b7-2e25f9bede1c.png)